docs/api/actions-and-connectors/create.asciidoc

```diff
@@ -99,6 +102,8 @@ authentication.
 `email`::
 (Required, string) The account email for HTTP Basic authentication.
 ====
+
+This object is not required for index or server log connectors.
```

Comment:
Could we add a link to the server log connector page here?

Reply:
The [Server log connector and action](https://kibana_134409.docs-preview.app.elstc.co/guide/en/kibana/master/server-log-action-type.html) page is quite sparse (and will become more so as I move all the reference and UI details out per https://github.com/elastic/response-ops-team/issues/58), so I think linking there is a bit of a dead end. If there's something there that you think is missing here, however, let me know and I can try to incorporate it.